packages/cozy-harvest-lib/src/models/ConnectionFlow.spec.js

```diff
@@ -481,6 +498,24 @@ describe('ConnectionFlow', () => {
     delete window.cozy
     delete window.ReactNativeWebView
   })
+
+  describe('contructor', () => {
+    beforeAll(() => {
+      watchKonnectorJob.mockReturnValue({on: () => ({})})
+    })
+
+    afterEach(() => {
+      jest.clearAllMocks()
+    })
+
+    afterAll(() => {
+      jest.restoreAllMocks()
```

Comment:
what is the point to use a clearAllMocks before restoreAllMocks:

https://jestjs.io/docs/mock-function-api#mockfnmockrestore

I read here that mockRestore does everything that mockReset does
And mockReset does everything that mockClear deos

Reply:
nit: i recommend to remove clearAllMocks

Reply:
I prefer to remove restoreAllMocks in this case. ClearAllMocks will erase the history of calls between test which is what I want in the first place.

And removing clearAllMocks makes the tests fail

Reply:
ok, then.

ps: I need to understand one day why mocks are not reset by default. 🤯

Reply:
Agreed